sqlmesh/core/loader.py

```diff
@@ -69,7 +69,9 @@ def update_model_schemas(dag: DAG[str], models: UniqueKeyDict[str, Model]) -> No
         if external:
             if "*" in model.columns_to_types:
                 raise ConfigError(
-                    f"Can't expand SELECT * expression for model '{name}'. Projections for models that use external sources must be specified explicitly at '{model._path}'"
+                    f"Can't expand SELECT * expression for model '{name}'."
+                    " Projections for models that use external sources must be specified explicitly"
```

Comment:
Perhaps include which command to run to fix the issue.

Reply:
I link to the external models docs page, which includes the CLI command. Do you think more is needed?

Reply:
Yeah, I think they should be able to try something immediately without having to read the lengthy doc page. If the quick way doesn't work for them, then they can check out the doc page.